utils.go

```diff
@@ -127,11 +128,25 @@ func structToMap(st any) (map[string]any, error) {
 	for i := 0; i < rv.NumField(); i++ {
 		fieldValue := rv.Field(i)
 		fieldType := rt.Field(i)
+
+		fieldTag := fieldType.Tag
+		tags, err := structtag.Parse(string(fieldTag))
+		if err != nil {
+			// failing to parse a struct tag means = tag is invalid = we should panic
+			panic(err)
+		}
+		var fieldName string
+		gtrsTag, err := tags.Get("gtrs")
```

Comment:
Isn't there a simple way in the standard lib to get the struct tag (without deps)? We don't have any format for now so we can just take the string

Reply:
I'm just not a fan of adding deps if we don't use any 'features'